operator-common/src/test/java/io/strimzi/operator/PlatformFeaturesAvailabilityTest.java

```diff
@@ -44,21 +44,21 @@ public static void after() {
 
     @Test
     public void networkPoliciesWithFancyCombinationTest() {
-        PlatformFeaturesAvailability pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_8);
+        PlatformFeaturesAvailability pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_15);
         assertThat(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), is(false));
-        pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_11);
-        assertThat(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), is(false));
-        pfa = new PlatformFeaturesAvailability(false, KubernetesVersion.V1_11);
+        pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_16);
+        assertThat(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), is(true));
+        pfa = new PlatformFeaturesAvailability(false, KubernetesVersion.V1_16);
         assertThat(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), is(true));
-        pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_12);
+        pfa = new PlatformFeaturesAvailability(true, KubernetesVersion.V1_17);
         assertThat(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), is(true));
     }
 
     @Test
-    public void testVersionDetectionOpenShift39(VertxTestContext context) throws InterruptedException {
+    public void testVersionDetectionOpenShift44(VertxTestContext context) throws InterruptedException {
         String version = "{\n" +
                 "  \"major\": \"1\",\n" +
-                "  \"minor\": \"9\",\n" +
+                "  \"minor\": \"17\",\n" +
```

Comment:
Hello @scholzj,  Here do we require to change the git version also?

Reply:
Yeah, I think you should replace it here as well.

Reply:
I have changed the minor version and kubernetes version. Iam not sure changing `gitVersion\": \"v1.9.1+a0ce1bc657\" `.  Like can we can change it to any new version ?

Reply:
If you want you can use this from my cluster:

```
{
  "major": "1",
  "minor": "20",
  "gitVersion": "v1.20.1",
  "gitCommit": "c4d752765b3bbac2237bf87cf0b1c2e307844666",
  "gitTreeState": "clean",
  "buildDate": "2020-12-18T12:00:47Z",
  "goVersion": "go1.15.5",
  "compiler": "gc",
  "platform": "linux/amd64"
}
```

Reply:
@scholzj thanks for helping me out. I have updated the PR with the required tests.